common/persistence/dataInterfaces.go

```diff
@@ -438,7 +438,7 @@ type (
 
 	// CreateWorkflowExecutionResponse is the response to CreateWorkflowExecutionRequest
 	CreateWorkflowExecutionResponse struct {
-		HistorySize int64
+		NewMutableStateStats MutableStateStatus
```

Comment:
if this is a "Status" type then the variable name "Stats" (which implies "statistics") seems wrong (or at least confusing to me)